integration-tests/features/stack_analyses.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Jobs API
+
+  Scenario: Check that the API entry point
+    Given System is running
+    When I access /api/stack-analyses
+    Then I should get 404 status code
+
+  Scenario: Check that the API entry point
+    Given System is running
+    When I wait 10 seconds
+    Then I wait 10 seconds
```

Comment:
I'm just curious, what's the purpose of this `When I wait ...` - `Then I wait ...` ?